src/index.ts

```diff
@@ -19,29 +19,37 @@ import {
   cancel,
   showCustomizationMessage,
 } from './commands';
-import { getConfig, findFiles } from './utils/vscode-extensions';
+import { getConfig, findFiles, asRelativePath } from './utils/vscode-extensions';
 
 function initialize(context: vscode.ExtensionContext) {
   const config = getConfig().vsicons;
   const settingsManager = new SettingsManager(vscode);
   registerCommands(context);
   manageWelcomeMessage(settingsManager);
   detectProject(findFiles, config)
-    .then((path) => {
-      if (path != null) {
-        const project = fs.readFileSync(path, "utf8");
-        const projectJson = (typeof project === "string") ? JSON.parse(project) : null;
-        if (projectJson) {
+    .then((results) => {
+      if (results != null && results.length) {
+        const isInRootFolder = asRelativePath(results[0].fsPath).includes('/');
```

Comment:
@robertohuertasm I can't reproduce your claim although I'm doing the reverse case. I'm starting the process in `D:` and opening a project that exists in `C:`. The result in both cases is `package.json`.

cwd: `"C:\Program Files (x86)\Microsoft VS Code Insiders"`
__dirname: `"d:\Projects\NodeJS\vscode-icons\out\src"`

And projects in either:
`"c:\Projects\test\package.json"` or `"d:\Projects\test\package.json"`